client/js/services/lb-services.js

```diff
@@ -1,3 +1,6 @@
+// Copyright IBM Corp. 2015,2016. All Rights Reserved.
+// Node module: loopback-getting-started-intermediate
+
```

Comment:
I think it's missing the text about `This file is licensed under the MIT License...`